### PR TITLE
Remove discord code.

### PR DESCRIPTION
**Pull request by wumingzhilian**
Remove discord code.

### CODE REVIEW

File: dojo_plugin/__init__.py [hunk 1]
```diff
@@ -22,7 +22,6 @@
 from .pages.desktop import desktop
 from .pages.users import users
 from .pages.settings import settings_override
-from .pages.discord import discord, maybe_award_belt
```
Comment (mudongliang) [inline dojo_plugin/__init__.py:25]: This maybe_award_belt is used in another site.

**Comment by mudongliang** [comment]
You should check all discord words, at least course.py and models/__init_.py

**Review by mudongliang — COMMENTED** [review]
discord_bot directory can be deleted. This is fine.

**Review by mudongliang — COMMENTED** [review]
pages/discord.py can be deleted, but be careful about the usage of these functions

**Comment by wumingzhilian** [comment]
image
local testing completed

**Pull request closed without merging**